virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -19,49 +26,78 @@ You can use any installation method, including user-provisioned, installer-provi
 If you install your cluster in xref:../../installing/installing-fips.adoc#installing-fips-mode_installing-fips[FIPS mode], no additional setup is required for {VirtProductName}.
 ////
 
-.IPv6
+// Section is in assembly so that we can use xrefs
+[id="virt-hardware-os-requirements_preparing-cluster-for-virt"]
+== Hardware and operating system requirements
 
-You cannot run {VirtProductName} on a single-stack IPv6 cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2193267[*BZ#2193267*])
+Review the following hardware and operating system requirements for {VirtProductName}.
 
-include::modules/virt-hardware-os-requirements.adoc[leveloffset=+1]
+[id="supported-platforms_preparing-cluster-for-virt"]
+=== Supported platforms
 
-* If your cluster uses worker nodes with different CPUs, live migration failures can occur because different CPUs have different capabilities. To avoid such failures, use CPUs with appropriate capacity for each node and set node affinity on your virtual machines to ensure successful migration. See xref:../../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Configuring a required node affinity rule] for more information.
+* On-premise bare metal servers. See xref:../../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#virt-planning-bare-metal-cluster-for-ocp-virt_preparing-to-install-on-bare-metal[Planning a bare metal cluster for {VirtProductName}].
+* Amazon Web Services bare metal instances. See link:https://access.redhat.com/articles/6409731[Deploy OpenShift Virtualization on AWS metal instance types].
 
-[role="_additional-resources"]
-.Additional resources
+ifdef::openshift-enterprise[]
+:FeatureName: Installing OpenShift Virtualization on AWS bare metal instances
+include::snippets/technology-preview.adoc[]
+:!FeatureName:
+endif::[]
```

Comment:
For the bullet points with TP blocks, I would nest the TP include under the relevant bullet point. For example:

```
* Amazon Web Services bare metal instances. See link:https://access.redhat.com/articles/6409731[Deploy OpenShift Virtualization on AWS metal instance types].
+
--
ifdef::openshift-enterprise[]
:FeatureName: Installing OpenShift Virtualization on AWS bare metal instances
include::snippets/technology-preview.adoc[]
:!FeatureName:
endif::[]
--

* IBM Cloud Bare Metal Servers. See link:https://access.redhat.com/articles/6738731[Deploy OpenShift Virtualization on IBM Cloud Bare Metal nodes].
+
--
ifdef::openshift-enterprise[]
:FeatureName: Installing OpenShift Virtualization on IBM Cloud Bare Metal Servers
include::snippets/technology-preview.adoc[]
:!FeatureName:
endif::[]
--
```

Reply:
Doesn't work. I tried that.

Reply:
Very cool! I will give that a try.